src/bulls_and_cows_lib/game.cpp

```diff
@@ -5,13 +5,164 @@
 #include "game_solver.hpp"
 #include "input.hpp"
 #include "main_menu.hpp"
+#include "random.hpp"
+#include <vector> 
 #include <chrono>
 #include <fstream>
 #include <iostream>
 #include <thread>
-
+using namespace std;
 namespace bulls_and_cows {
 
+	int distinct(char rand, std::vector <char> secretCode)
+	{
+		int vrai = 0;
+		for (int i = 0; i < secretCode.size(); i++)
```

Comment:
Quand on fait une boucle sur un container (une collection) et qu'on n'a pas besoin de l'index de l'élément courant, mais seulement de l'élément courant lui-meme, il est préférable d'utiliser une boucle _range-for_ (https://en.cppreference.com/w/cpp/language/range-for).
Exemple :
for (const char c : secretCode)
{
    if (rand == c)
        vrai = 1;
}